ompi/class/ompi_seq_tracker.c

```diff
@@ -21,178 +21,163 @@
 
 #include "ompi/class/ompi_seq_tracker.h"
 
+OBJ_CLASS_INSTANCE(ompi_seq_tracker_range_t, opal_list_item_t, NULL, NULL);
 
-
-OBJ_CLASS_INSTANCE(ompi_seq_tracker_range_t,
-                   opal_list_item_t,
-                   NULL,
-                   NULL);
-
-
-static void ompi_seq_tracker_construct(ompi_seq_tracker_t* seq_tracker) {
+static void ompi_seq_tracker_construct(ompi_seq_tracker_t *seq_tracker)
+{
     OBJ_CONSTRUCT(&seq_tracker->seq_ids, opal_list_t);
-    seq_tracker->seq_ids_current = (ompi_seq_tracker_range_t*)opal_list_get_end(&seq_tracker->seq_ids);
+    seq_tracker->seq_ids_current = (ompi_seq_tracker_range_t *) opal_list_get_end(
+        &seq_tracker->seq_ids);
 }
 
-
-static void ompi_seq_tracker_destruct(ompi_seq_tracker_t* seq_tracker)
+static void ompi_seq_tracker_destruct(ompi_seq_tracker_t *seq_tracker)
 {
-    opal_list_item_t* item;
-    while(NULL != (item = opal_list_remove_first(&seq_tracker->seq_ids)))
+    opal_list_item_t *item;
+    while (NULL != (item = opal_list_remove_first(&seq_tracker->seq_ids)))
         OBJ_RELEASE(item);
     OBJ_DESTRUCT(&seq_tracker->seq_ids);
 }
 
-OBJ_CLASS_INSTANCE(
-                   ompi_seq_tracker_t,
-                   opal_object_t,
-                   ompi_seq_tracker_construct,
+OBJ_CLASS_INSTANCE(ompi_seq_tracker_t, opal_object_t, ompi_seq_tracker_construct,
                    ompi_seq_tracker_destruct);
 
-
 /**
  *  Look for duplicate sequence number in current range.
  *  Must be called w/ matching lock held.
  */
 
-bool ompi_seq_tracker_check_duplicate(
-    ompi_seq_tracker_t* seq_tracker,
-    uint32_t seq_id)
+bool ompi_seq_tracker_check_duplicate(ompi_seq_tracker_t *seq_tracker, uint32_t seq_id)
 {
-    ompi_seq_tracker_range_t* item;
-    const ompi_seq_tracker_range_t* sentinel = (ompi_seq_tracker_range_t*)opal_list_get_end(&seq_tracker->seq_ids);
+    ompi_seq_tracker_range_t *item;
+    const ompi_seq_tracker_range_t *sentinel = (ompi_seq_tracker_range_t *) opal_list_get_end(
```

Comment:
we've historically broken this at the assignment, but I'm not sure I have huge feelings.

Reply:
Ok. I made the penalty for breaking before the first argument bigger than breaking on assignment. This will be fixed in the next push.